include/fiction/algorithms/verification/virtual_miter.hpp

```diff
@@ -0,0 +1,121 @@
+//
+// Created by benjamin on 7/31/24.
+//
+
+#ifndef FICTION_VIRTUAL_MITER_HPP
+#define FICTION_VIRTUAL_MITER_HPP
+
+#include "fiction/networks/virtual_pi_network.hpp"
+
+#include <mockturtle/algorithms/cleanup.hpp>
+#include <mockturtle/traits.hpp>
+
+#include <algorithm>
+#include <optional>
+
+namespace fiction
+{
+
+/*! \brief Creates a combinational miter from two networks.
+ *
+ * This method combines two networks that have the same number of primary
+ * inputs and the same number of primary outputs into a miter.  The miter
+ * has the same number of inputs and one primary output.  This output is the
+ * OR of XORs of all primary output pairs.  In other words, the miter outputs
+ * 1 for all input assignments in which the two input networks differ.
+ *
+ * All networks may have different types.  The method returns an optional, which
+ * is `nullopt`, whenever the two input networks don't match in their number of
+ * primary inputs and primary outputs.
```

Comment:
This does not discuss how this function differs from `mockturtle::miter`. Please mention virtual PIs here.